### PR TITLE
Led control issue

### LINKED ISSUE #144: LED register is not working very well on XL-320
There is always some odd things with led register on the poppy-ergo-jr (it is a bug of Dxl320IO). 
@Tibo42 could you detail a protocol to reproduce the bug ?
@pierre-rouanet don't you think it should be solved for the next release ?

I checked [dxl_to_led_color](https://github.com/poppy-project/pypot/blob/master/pypot/dynamixel/conversion.py#L256), [led_color_to_dxl](https://github.com/poppy-project/pypot/blob/master/pypot/dynamixel/conversion.py#L260), and all the table of [DXL320IO registers](https://github.com/poppy-project/pypot/blob/master/pypot/dynamixel/io/io_320.py), and I didn't see anything wrong...

### PR DESCRIPTION
**Pull request by pierre-rouanet**
WIP: This PR tries to fix #144 and #139 

I fix a tricky indentation issue that prevents the teardown of a primitive to be correctly synced before stopping it. It seems to have fixed the led issues with the Dance primitive for instance where green LEDs were staying on after the end of the dance.

### CODE REVIEW

File: pypot/dynamixel/io/io.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import itertools
+
 from .abstract_io import (AbstractDxlIO, _DxlControl,
                           _DxlAccess, DxlTimeoutError)
-from ..protocol import v1 as v1
 from .. import conversion as conv
+from ..protocol import v1 as v1
```
Comment (show0k) [inline pypot/dynamixel/io/io.py]: Does it change really something ?

**Comment by pierre-rouanet** [comment]
On my side, the "LED part" is ready. @show0k I missed something or should I merge?

I must confess, I'm quite keen to move quickly towards this 2.11 release :smile:

**Comment by show0k** [comment]
Ok, I didn't check anything new from my side as I have no XL320 with me, but go on !

**Comment by pierre-rouanet** [comment]
We've done some tests with @Tibo42. It seems ok!

**Pull request merged**